### DISCUSSION TITLE
Pimcore Hosting and Deployment in 2021

### DISCUSSION THREAD

**Discussion by mittererr** [pimcore/pimcore#7823] [General]
Where are you hosting your Pimcore projects? Any recommendations? Which deployment strategies are you using with your hosting?

At the moment I'm using a DomainFactory Host L7 and Deployer for deployment but there are problems with missing PHP modules and so on. So I'm searching vor something more professional.

Thx

**Comment by passioneight** [comment]
Any host will do just fine as long as you can ensure the setup as described in [Pimcore's documentation regarding system requirements](https://pimcore.com/docs/pimcore/current/Development_Documentation/Installation_and_Upgrade/System_Requirements.html).

As for the deployment: it variies greatly. Some just upload files manually, which works but is pretty old school and prone to error. Others create their own system to deploy everything, probably relying on their VCS of choice to handle most of the deployment. Others use [`CI/CD`](https://www.atlassian.com/continuous-delivery/continuous-deployment).

The decision, however, is yours to make.
